i18n/en.json

```diff
@@ -4059,7 +4059,7 @@
   "multiselect.selectChannels": "Use ↑↓ to browse, ↵ to select.",
   "multiselect.selectTeams": "Use ↑↓ to browse, ↵ to select.",
   "navbar_dropdown.about": "About {appTitle}",
-  "navbar_dropdown.accountSettings": "Profile",
+  "navbar_dropdown.accountSettings": "Settings",
```

Comment:
I'm just talking with the Docs team to figure out if we are better off creating a new key or if updating this value is ok. Everything else looks good though!

Reply:
Yes, I wasn't sure about this as well, but it seemed more appropriate this way (variable name wise at least). So whatever you decide, I'll change the PR to that!